test/plugin/constants.rb

```diff
@@ -509,7 +517,7 @@ module Constants
       type: EC2_CONSTANTS[:resource_type],
       labels: {
         'instance_id' => EC2_VM_ID,
-        'region' => EC2_PREFIXED_ZONE,
+        'region' => EC2_PREFIXED_REGION,
```

Comment:
Should we rename `EC2_PARAMS` to `EC2_REGION_PARAMS`? It's a bit confusing for  `EC2_PARAMS` to use `region` while the default behavior (if the config is not set) still uses `availability zone`.